index.html

```diff
@@ -6,23 +6,58 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Tracom Academy website</title>
-</head>
-
-<body>
-    <header class="main-header">
-        <nav class="main-nav">
-            <img src="Assets/tracom_services_cover.jpg" alt="tracom-logo" class="main-logo">
-            <ul id="main-nav-ul">
-                <li><a href="#">Home</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Services</a></li>
-                <li><a href="#">Team</a></li>
-                <li><a href="#">Contact</a></li>
-            </ul>
-        </nav>
-    </header>
-
-</body>
-
-
-</html>
\ No newline at end of file
+    </head>
+    <body>
+        <header class="main-header">
+            <nav class="main-nav">
+                <img src="Assets/tracom_services_cover.jpg" alt="tracom-logo" class="main-logo" >
+                <ul id="main-nav-ul">
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Services</a></li>
+                    <li><a href="#">Team</a></li>
+                </ul>
+            </nav>            
+        </header>
+     
+        <div class="Contact">
```

Comment:
center the contact container

Reply:
you can also try to add some few styles to add some life to it like adding a backgroud image and attractive colors